apps/experiments/views/experiment.py

```diff
@@ -291,7 +292,11 @@ def create_channel(request, team_slug: str, experiment_id: int):
         except ChannelAlreadyUtilizedException as exception:
             messages.error(request, exception.html_message)
             return redirect("experiments:single_experiment_home", team_slug, experiment_id)
-
+        if platform == ChannelPlatform.WHATSAPP:
+            webhook_url = absolute_url(
+                reverse("channels:new_turn_message", kwargs={"experiment_id": experiment.public_id}), is_secure=True
+            )
+            messages.info(request, f"Use the following URL when setting up the webhook in Turn.io: {webhook_url}")
```

Comment:
What do you think about putting this in the `extra_form`? Then the view code would become:

```python
if extra_form:
    message = extra_form.get_message(experiment)
    if message:
        messages.info(request, message)
```

Reply:
like this? https://github.com/dimagi/open-chat-studio/pull/223/commits/19e6b04b03dd9d05556acf9759d536f4cb04e4ad